CONTRIBUTING.md

```diff
@@ -117,6 +117,78 @@ You can learn more about [pull requests on GitHub](https://help.github.com/en/gi
+
+Before you start making changes to the files, make a branch to commit all of your changes to. This will help keep your work organized, and will help you avoid committing undesired changes to your main branch.
+To do this in VSCode, follow the steps below.
+
+1. Ensure you are in the `ipfs-docs` directory on VSCode.
+2. Click the branch icon in the bottom left corner of the VSCode window.
+ ![Create a new branch](./images/create-new-branch.png)
+1. A text field will open at the top of the window, type the desired name of your new branch. Preferably, the name of the branch reflects the work that you will be doing on the branch.
+1. Once you have the name figured out, click **+Create new branch**.
+
+Make sure that any changes you make are done in this new branch, you can verify this by checking the current branch in the bottom left corner of the window.
+
+### Adding a single page
+
+Adding a single page to an already existing section is as simple as adding one line to the `config.js` file.
+
+When adding the new line, be sure to follow the exact same format and syntax as the other lines that are already present.
+
+1. Open `config.js` in VSCode.
+1. Find the section you want to add a new page to. For this example, a page titled **Add a new page** will be added to the **Basics** section.
+ ![Add a new page](./images/add-new-page.png)
+1. Add the new line to the desired section.
+  ```
+  'title-of-section/title-of-new-page'
+  ```
+4. With the page now added to `config.js`, save the file, stage the changes, and commit them to your branch.
+
+### Adding a new section
+
+Adding a new section to the sidebar of the docs is a little more in-depth. For this example, a new section titled **Test** will be added between the **Basics** and **Concepts** sections.
+
+When adding a new section, be sure you have the opening and closing curly brackets (`{` `}`) in the correct column. The easiest way to do this is to follow the same positioning and format as the other sections that are already in the file.
+
+1. Open `config.js` in VSCode.
+1. Find the spot that you wish to add a new section. Again, for this example, the spot is between the **Basics** and **Concepts** sections.
+ ![Basics and concepts sections](./images/basics-and-concepts-sections.png)
+3. Once you have found the right spot, add in the following lines, replacing any variables with your own content.
+ ![Add a new section](./images/add-new-section.png)
+
+### Commit your changes
+
+Now that you have made the changes to `config.js`, you can commit them to your branch and make a pull request.
+
+1. Save the changes you have made to the `config.js` file.
+1. Click the **source control** button in the VSCode sidebar.
+ ![Source control](./images/source-control.png)
+2. In the **source control** menu, mouse-over the changes you made and click the **+** button to stage those changes.
```

Comment:
I think "mouse-over" should not have a hyphen - haven't confirmed this though

Reply:
As far as I can tell, mouseover is, in fact, one word.